lib/stdlib/src/edlin_type_suggestion.erl

```diff
@@ -220,10 +220,19 @@ simplified_type(file, name_all, 0) -> {type, erlang, string, []};
 simplified_type(file, name, 0) -> {type, erlang, string, []};
 simplified_type(_Module, _TypeName, _Arity) -> none.
 
+code_get_doc_cache(Mod) ->
+    case get(Mod) of
```

Comment:
```suggestion
    case get({?MODULE, Mod}) of
```

I suggest that we tag the value with something so that we reduce the chances of colliding with other code.